src/components/pages/error-page/errorcode-content/search-form/SearchForm.tsx

```diff
@@ -1,12 +1,10 @@
 import React, { useState } from 'react';
-import { TextField } from '@navikt/ds-react';
-import { ClearIcon } from 'components/pages/error-page/errorcode-content/clear-icon/ClearIcon';
-import { Button } from 'components/_common/button/Button';
+import { Search } from '@navikt/ds-react';
 
 import style from './SearchForm.module.scss';
 
-const MAX_SEARCH_LENGTH = 200;
-const HEADER_TEXT = 'Hva leter du etter?';
+const HEADER_TEXT = 'Du kan søke:';
```

Comment:
Bare flisespikking, men kanskje vi bare kan sette disse rett inn i Search-komponenten siden de bare brukes det ene stedet? Mest for lesbarhet i kontekst altså, og kanskje bare smak og behag.